server/plugin_test.go

```diff
@@ -64,23 +63,76 @@ func setupTestPlugin(t *testing.T, api *plugintest.API, siteURL string) *Matterp
 		},
 	}
 	p.SetAPI(api)
-	err := p.OnActivate()
-	require.Nil(t, err)
+	p.router = p.InitAPI()
+
 	return p
 }
 
 func TestPluginOnActivate(t *testing.T) {
-	p := &MatterpollPlugin{
-		Config: &Config{},
+	t.Run("empty config", func(t *testing.T) {
+		api := &plugintest.API{}
+		api.On("GetServerVersion").Return("5.4.0")
+		p := &MatterpollPlugin{}
+		p.SetAPI(api)
+
+		err := p.OnActivate()
+
+		assert.NotNil(t, err)
+	})
+
+	for name, test := range map[string]struct {
+		SetupAPI    func(*plugintest.API) *plugintest.API
+		ShouldError bool
+	}{
+		"server version: 5.5.0": {
```

Comment:
Must we modify these test case whenever updating minimumVersion?

If so, it had better to fix `name`s to `greater majaor version than minimumVersion`, `same version as minimumVersion`, and so on.
And add cases about `minor version` and `-rc` versions

Reply:
While lookinginto the server code, I found about that `GetServerVersion` acutely doesn't return information about the rc. So no test case is needed.

I updated the tests to be relative to `minimumServerVersion`.